pkg/jx/cmd/install.go

```diff
@@ -381,151 +383,229 @@ func (options *InstallOptions) Run() error {
-			confirm := &survey.Confirm{
-				Message: "Uninstalling existing tiller binary:",
-				Default: true,
-			}
-			flag := true
-			err = survey.AskOne(confirm, &flag, nil)
-			if err != nil || flag == false {
-				return errors.New("Existing tiller must be uninstalled first in order to use the jx in tiller less mode")
-			}
-			// Uninstall helm and tiller first to avoid using some older version
-			err = options.UninstallBinary(binDir, "tiller")
-			if err != nil {
-				return errors.Wrap(err, "uninstalling existing tiller binary")
-			}
-		}
+	err = options.configureHelm3(ns)
+	if err != nil {
+		return errors.Wrap(err, "configuring helm3")
+	}
 
-		_, install, err = shouldInstallBinary(helmBinary)
-		if !install && err == nil {
-			confirm := &survey.Confirm{
-				Message: "Uninstalling existing helm binary:",
-				Default: true,
-			}
-			flag := true
-			err = survey.AskOne(confirm, &flag, nil)
-			if err != nil || flag == false {
-				return errors.New("Existing helm must be uninstalled first in order to use the jx in tiller less mode")
-			}
-			// Uninstall helm and tiller first to avoid using some older version
-			err = options.UninstallBinary(binDir, helmBinary)
-			if err != nil {
-				return errors.Wrap(err, "uninstalling existing helm binary")
-			}
+	err = options.installAddons()
+	if err != nil {
+		return errors.Wrap(err, "installing the Jenkins X Addons")
+	}
+
+	options.logAdminPassword()
+
+	err = options.configureJenkins(ns)
+	if err != nil {
+		return errors.Wrap(err, "configuring Jenkins")
+	}
+
+	err = options.createEnvironments(ns)
+	if err != nil {
+		return errors.Wrap(err, "creating the environments")
+	}
+
+	err = options.saveChartmuseumAuthConfig()
+	if err != nil {
+		return errors.Wrap(err, "saving the Chartmuseum auth configuration")
+	}
+
+	if options.Flags.RegisterLocalHelmRepo {
+		err = options.registerLocalHelmRepo(options.Flags.LocalHelmRepoName, ns)
+		if err != nil {
+			return errors.Wrapf(err, "registering the local helm repo '%s'", options.Flags.LocalHelmRepoName)
 		}
+	}
 
-		dependencies = append(dependencies, "tiller")
-		options.Helm().SetHost(tillerAddress())
+	err = options.generateGitOpsDevEnvironmentConfig(gitOpsDir)
+	if err != nil {
+		return errors.Wrap(err, "generating the GitOps development environment config")
 	}
-	dependencies = append(dependencies, helmBinary)
-	err = options.installRequirements(options.Flags.Provider, dependencies...)
+
+	err = options.applyGitOpsDevEnvironmentConfig(gitOpsEnvDir, ns)
 	if err != nil {
-		return errors.Wrap(err, "failed to install the platform requirements")
+		return errors.Wrap(err, "applying the GitOps development environment config")
 	}
 
+	log.Success("\nJenkins X installation completed successfully\n")
+
+	options.logAdminPassword()
+
+	log.Infof("\nYour Kubernetes context is now set to the namespace: %s \n", util.ColorInfo(ns))
+	log.Infof("To switch back to your original namespace use: %s\n", util.ColorInfo("jx ns "+originalNs))
```

Comment:
for documentation purposes, we should use the fully-named `jx namespace` rather than the shorthand `jx ns`